CHANGELOG.md

```diff
@@ -1 +1,11 @@
+- CloudSQL instances created with `firebase dataconnect:connect` now default to Postgres 17.
+- Fixed an issue with deploying indexes to Firestore Enterprise edition databases where explicit `__name__` fields could be incorrectly handled.
+- The `experimental:mcp` command has been renamed to `mcp`. The old name is now an alias.
+- `firebase_update_environment` MCP tool supports accepting Gemini in Firebase Terms of Service.
+- Fixed a bug when `firebase init dataconnect` failed to create a React app when launched from VS Code extension (#9171).
+- Improved the clarity of the `firebase apptesting:execute` command when you have zero or multiple apps.
+- `firebase dataconnect:sql:migrate` now supports Cloud SQL instances with only private IPs. The command must be run in the same VPC of the instance to work. (##9200)
+- Fixed an issue where `firebase deploy --only firestore` would fail with 403 errors on projects that never had a database created.
+- Added `remoteconfig:experiments:get`, `remoteconfig:experiments:list`, and `remoteconfig:experiments:delete` commands to manage Remote Config experiments.
```

Comment:
Looks like this CHANGELOG is out of date and is pulling in things that went out in the last release. Can you change this to only include lines 9, 10, and 11?